### DISCUSSION TITLE
Backlinks in Zettlr 2

### DISCUSSION THREAD

**Discussion by mikae1** [Zettlr/Zettlr#2867] [General]
Hi! I came across [this old post in the Zettlr forum](https://forum.zettlr.com/discussion/comment/1206/#Comment_1206) mentioning that

> I think some sort backlinking (like Obsidian) is due in version 2.0

I have trouble finding any info on this. Did this feature ever make it to Zettlr 2?

**Comment by nathanlesage** [comment]
It didn't make it into 2.0, but it's on the roadmap still. However, I just counted 67 open bugs, and those have precedence :D

**Reply by mikae1** [reply]
Oh! Perhaps for 4.0 then 😀
